dashboard/src/components/Tabs/Builds/BuildCards.tsx

```diff
@@ -5,21 +5,19 @@ import { memo, useMemo } from 'react';
 import BaseCard from '@/components/Cards/BaseCard';
 import { GroupedTestStatus } from '@/components/Status/Status';
 
-import type { ISummaryTable } from '@/components/Summary/Summary';
-import { DumbSummary, SummaryItem } from '@/components/Summary/Summary';
-import type { TFilterObjectsKeys } from '@/types/tree/TreeDetails';
+import type { ISummaryTable } from '../Summary';
+import { DumbSummary, MemoizedSummaryItem } from '../Summary';
 
-interface IErrorsSummaryBuild extends Pick<ISummaryTable, 'summaryBody'> {
-  toggleFilterBySection: (
-    value: string,
-    filterSection: TFilterObjectsKeys,
-  ) => void;
+interface IErrorsSummaryBuild<T> extends Pick<ISummaryTable, 'summaryBody'> {
```

Comment:
I couldn't fin a case where `T` was not `TFilterObjectKeys` this does not need to be a generic